### PR TITLE
Obstacles

### PR DESCRIPTION
**Pull request by bhasseman**
This branch implemented the two obstacle classes, Block and Statue, which can be drawn and shown in the game as needed. A spritefactory, ObstacleTextureStorage, was created to make the loading of the textures easier. I added the drawing of the block object to Game1, similar to the example picture from the Sprint 2 description from Boggus' site. I also edited the IObstacle interface to make the implementation of the Draw method easier. I removed the Yoshi spritesheet from the Content folder as it was no longer necessary. Lastly, I slightly modified Game1.cs to get rid of the drawing of the credits.

### CODE REVIEW

File: CrossPlatformDesktopProject/CrossPlatformDesktopProject/CrossPlatformDesktopProject.csproj [hunk 1]
```diff
@@ -1,138 +1,141 @@
...
+    <Compile Include="ISpriteFactory.cs" />
+    <Compile Include="IWorldItem.cs" />
+    <Compile Include="Controllers\MouseController.cs" />
+    <Compile Include="Controllers\KeyController.cs" />
+    <Compile Include="Game1.cs" />
+    <Compile Include="IController.cs" />
+    <Compile Include="Link\LinkIdleEast.cs" />
+    <Compile Include="Link\LinkIdleSouth.cs" />
+    <Compile Include="Link\LinkIdleNorth.cs" />
+    <Compile Include="Link\ObstacleTextureStorage.cs" />
```
Comment (sweeneyde) [inline CrossPlatformDesktopProject/CrossPlatformDesktopProject/CrossPlatformDesktopProject.csproj]:
```suggestion
    <Compile Include="Obstacles\ObstacleTextureStorage.cs" />
```

Comment (sweeneyde) [reply CrossPlatformDesktopProject/CrossPlatformDesktopProject/CrossPlatformDesktopProject.csproj]: It won't build because the file moved.

**Review by sweeneyde — CHANGES_REQUESTED** [review]

**Comment by sweeneyde** [comment]
With that one change, it builds on my machine and I see the block.

**Comment by bhasseman** [comment]
> With that one change, it builds on my machine and I see the block.

Changed as per your suggestion and builds on mine as well.

**Pull request merged**